doc/source/coding-style/pep8.rst

```diff
@@ -259,30 +179,35 @@ from PyPi.
 
 .. code::
 
-   pip install package
+   python -m pip install package
 
 
-Class Naming Conventions
-~~~~~~~~~~~~~~~~~~~~~~~~
-Use `camel case <https://en.wikipedia.org/wiki/Camel_case>`_ when naming classes. Do not separate words
-with underscores. 
+Classes
+~~~~~~~
+Use `camel case <https://en.wikipedia.org/wiki/Camel_case>`_ when naming
+classes. Do not separate words with underscores. 
 
 .. code:: python
 
    class MyClass():
        """Docstring for MyClass"""
-       pass
+       ...
+
 
+Use a lowercase word or words for Python functions or methods. Separate words
```

Comment:
I don't know what I did here to get this color change--but the wording here is fine! Sorry!